lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -48,7 +75,7 @@
     <button class="action action-cancel">
         <%=
             interpolate_text(
-                _.escape(gettext("Cancel {span_start}a new team{span_end}")),
+                _.escape(gettext("Cancel {span_start} team {span_end}")),
```

Comment:
@muzaffaryousaf I think we need a verb here to make it clear what action is being cancelled. If we can make the distinction between "create" and "update" can we make it "Cancel creating team" and "Cancel updating team"? If we can't make the distinction, then "Cancel creating or updating team"? @clrux any thoughts?